### PR TITLE
Feature  #164489786 Get All articles

### PR DESCRIPTION
**Pull request by mystere10**
**What does this PR do?**
- [x] This PR implements the get all articles from the database

**Description of Task to be completed?**
During this task, the get all article feature was implemented by
- Creating the get all article route
- Creating the get all article method in the article class
- Implementing the findAll method in the article model
- Testing the get all article endpoint

**How should this be manually tested?**
This endpoint can be tested by writing the http://localhost:3000/api/articles/ in the URL of a browser or use postman to test.

**What are the relevant pivotal tracker stories?**
[Feature #164489786]

**Screenshot**
Screen Shot 2019-04-18 at 20 51 19

### CODE REVIEW

**Review by frankhn — APPROVED** [review]
LGTM

File: controllers/article.js [hunk 1]
```diff
@@ -13,23 +13,47 @@ class Article {
    * @returns {Object} Article
    */
   static async createArticle(req, res) {
+    const {
+      title, body, taglist, authorid
```
Comment (frankhn) [inline controllers/article.js]: I don't think you should be getting author id from the body?

File: test/article.js [hunk 2]
```diff
@@ -115,3 +119,40 @@ describe('Test the title', () => {
...
+  });
+});
+describe('Test all articles', () => {
+  it('should return all the articles', () => {
+    chai.request(index).get('/api/articles').then((res) => {
+      res.should.have.status(200);
+      res.body.should.be.a('object');
+    })
+      .catch((err) => {
+        console.log(err);
```
[the remark this answers is not in the document]
Comment (frankhn) [reply test/article.js:154]: remove the console.

**Review by frankhn — CHANGES_REQUESTED** [review]

File: controllers/article.js (hunk 1 above)
Comment (mystere10) [reply controllers/article.js]: Yes, this was changed in later PR, as part of continuous integration.

**Pull request merged**